app/src/ui/publish-repository/publish-repository.tsx

```diff
@@ -90,14 +99,27 @@ export class PublishRepository extends React.Component<
     this.updateSettings({ private: event.currentTarget.checked })
   }
 
-  private onOrgChange = (event: React.FormEvent<HTMLSelectElement>) => {
+  private onOrgChange = async (event: React.FormEvent<HTMLSelectElement>) => {
     const value = event.currentTarget.value
     const index = parseInt(value, 10)
     if (index < 0 || isNaN(index)) {
-      this.updateSettings({ org: null })
+      this.updateSettings({
+        org: null,
+        canCreatePrivateRepo: canCreatePrivateRepo(this.props.account),
```

Comment:
If `canCreatePrivateRepo` and `private` are set to the same value, why do we need two different properties?